FF1Blazorizer/wwwroot/index.html

```diff
@@ -49,5 +49,10 @@
     </app>
     <script src="_framework/blazor.webassembly.js" autostart="false"></script>
     <script src="js/site.js"></script>
+
+    <!-- This iframe is a hidden stub page that we will interact with to store preferences at
+         a single domain, to avoid having separate preferences/saved ROM/presets on every
+         instanced site version.  Where we load this from determines that domain. -->
+    <iframe src="preferences.html" height="0" width="0"></iframe>

```

Comment:
wouldnt this be `<site>.finalfantasyrandomizer.com/preferences.html`, where <site> will be different for each instanced site? How does this allow for sharing localStorage? Or am I missing something obvious?

Reply:
Yea, that's a mistake, it is supposed to be `src="https://beta.finalfantasyrandomizer.com/preferences.html"`

Reply:
Fixed.